sim/druid/swipe.go

```diff
@@ -39,18 +26,19 @@ func (druid *Druid) registerSwipeBearSpell() {
 	switch druid.Ranged().ID {
 	case IdolOfBrutality:
 		rageCost -= 3
+	case IdolOfUrsinPower:
+		baseMultiplier += .03
 	}
+	rageMetrics := druid.NewRageMetrics(core.ActionID{SpellID: 431446})
 
 	druid.SwipeBear = druid.RegisterSpell(Bear, core.SpellConfig{
-		ActionID:    core.ActionID{SpellID: spellID},
+		ActionID:    core.ActionID{SpellID: 9908},
```

Comment:
Looks like swipe has 5 ranks. Can you split out the damage / actionID by level here?